cogs/covidcog.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord.ext import commands
+import requests
+
+
+def embedCreator(data):
+    state = data["state"]
+    embedTitle = ":world_map: Covid Stats in " + state + " :world_map:"
+    firstField = {
+        "name": "Total " + state + " Cases",
+        "value": "Total Cases: {:,}".format(data["cases"]),
+        "inline": True
+    }
+    secondField = {
+        "name": "Total " + state + " Deaths",
+        "value": "Total Deaths: {:,}".format(data["deaths"]),
+        "inline": True
+    }
+    thirdField = {
+        "name": "Total " + state + " Tests",
+        "value": "Total Tests: {:,}".format(data["tests"]),
+        "inline": True
+    }
+    embed = discord.Embed(title=embedTitle, color=0x8D0000)
+    embed.description = "Statistics for Covid-19 in " + state + "\n"
+    embed.add_field(
+        name=firstField["name"], value=firstField["value"], inline=firstField["inline"])
+    embed.add_field(
+        name=secondField["name"], value=secondField["value"], inline=secondField["inline"])
+    embed.add_field(
+        name=thirdField["name"], value=thirdField["value"], inline=thirdField["inline"])
+    embed.set_footer(text="Data from corona.lmao.ninja")
+
+    return embed
+
+
+class CovidCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def covid(self, ctx):
+        r = requests.get("https://corona.lmao.ninja/v2/states")
+        apidata = r.json()
+        state = ctx.message.content[7:].title()
```

Comment:
You don't need to manually parse the argument with `[7:]`, you can make the argument a parameter of the command's method, e.g.
```python
@commands.command()
async def covid(self, ctx, state)
    # Your code ...
    state = state.title()
```

This is cleaner and more readable. It will require the user to enter the name of a state whose name contains more than one word with quotes, but this is not too much of a problem.